pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -147,8 +157,32 @@ func (c *Controller) logPacket(pktIn *ofctrl.PacketIn) error {
 		return fmt.Errorf("received error while handling packetin for NetworkPolicy: %v", err)
 	}
 
-	// Store log file
-	AntreaPolicyLogger.Printf("%s %s %s %s SRC: %s DEST: %s %d %s", ob.tableName, ob.npRef, ob.disposition, ob.ofPriority, ob.srcIP, ob.destIP, ob.pktLength, ob.protocolStr)
+	// Create 1 sec buffers for deduplicating non-Allow packet log.
+	logMsg := fmt.Sprintf("%s %s %s %s SRC: %s DEST: %s %d %s", ob.tableName, ob.npRef, ob.disposition, ob.ofPriority, ob.srcIP, ob.destIP, ob.pktLength, ob.protocolStr)
+	if ob.disposition == openflow.DispositionToString[openflow.DispositionAllow] {
+		AntreaPolicyLogger.Printf(logMsg)
+	} else {
+		_, ok := logDeduplication[logMsg]
+		// Increase count and renew buffer if duplicated within 1 sec, create buffer otherwise.
+		if ok {
+			logDeduplication[logMsg].count++
+			logDeduplication[logMsg].bufferTimer.Reset(time.Second)
+		} else {
+			record := logDedup{1, logMsg, time.Now(), time.NewTimer(time.Second)}
+			logDeduplication[logMsg] = &record
+
+			// Go routine for logging when buffer timer stops.
+			go func(r *logDedup) {
+				<-r.bufferTimer.C
+				if r.count == 1 {
+					AntreaPolicyLogger.Printf(logMsg)
+				} else {
+					AntreaPolicyLogger.Printf("<-%d %s-> %s", r.count, time.Since(r.initTime), r.dupLogInfo)
```

Comment:
Not sure `<-10 1.2s->` is the best way to express that we've seen 10 dup entries in 1.2 seconds.... we could be a bit more verbose here in my opinion

Reply:
Thanks, any suggestion on verbose, perhaps sth. like `received 10 in 1.2s`?

Reply:
I would say something like [10 packets in 1.2s] but I'll let others comment too